### PR TITLE
[APIP-251] Cleanup Release Log

### PR DESCRIPTION
**Pull request by li7vinov-denis**
https://pandaflow.atlassian.net/browse/APIP-251

### CODE REVIEW

**Review by ericpopivker — CHANGES_REQUESTED** [review]
What about other changes and improvements we plan to push?
Like new Etsy validaton, etc...

**Comment by li7vinov-denis** [comment]
@ericpopivker, added one more change for wrong http codes

before:
127 0 0 1_4000_release-log

now:
127 0 0 1_4000_release-log (1)

File: release-log.md [hunk 1]
```diff
@@ -4,76 +4,102 @@ layout: default
 nav_order: 3
 ---
 
-# 23 Aug 2024
+### Release 0.23
```
Comment (ericpopivker) [inline release-log.md]: Can we change it to 1.23
Otherwise it seems like it is beta.

**Review by ericpopivker — CHANGES_REQUESTED** [review]
Can we

File: release-log.md (hunk 1 above)
Comment (li7vinov-denis) [reply release-log.md]: updated for both releases

**Review by ericpopivker — APPROVED** [review]

**Pull request merged**